hack/update-codegen.sh

```diff
@@ -34,11 +34,11 @@ echo "" > /tmp/headerfile
 # Makefile's value and also force mod=vendor here
 export "GOFLAGS=-mod=vendor"
 
-echodate "Generating kubermatic:v1"
+echodate "Generating kubermatic:v1 and app.kubermatic:v1"
 ./vendor/k8s.io/code-generator/generate-groups.sh all \
   k8c.io/kubermatic/v2/pkg/crd/client \
   k8c.io/kubermatic/v2/pkg/crd \
-  "kubermatic:v1" \
+  "kubermatic:v1 app.kubermatic:v1" \
```

Comment:
```suggestion
  "kubermatic:v1" \
```

let's leave old stuff out

Reply:
removed :)